fares-data-build-tool/fdbt-site
# Feature/tfn 453 define user type

# Description

-   This PR introduces a new page to allow the user to define their passenger type if they have previously select any user type other than 'Any' on the previous page. The page contains two distinct questions for the user which make use of conditionally revealing inputs (one with text inputs and the other with checkboxes). 

# Testing instructions

-   Pull down this branch and run the site locally.
-   Run through the user journey from start to finish selecting (a) 'Any' on the passengerType page and (b) anything other than 'Any' on the passengerType page.
-   Check that the definePassengerType page carries user input in the cookie and displays errors to the user correctly.
-   Check that the final json in S3 contains information entered by the user on this page.

# Type of change


# Checklist:


[definePassengerType.pdf](https://github.com/fares-data-build-tool/fdbt-site/files/4689848/definePassengerType.pdf)
